database/postgres/event.go

```diff
@@ -37,7 +37,8 @@ const (
     raw, data, headers, is_duplicate_event,
 	COALESCE(source_id, '') AS source_id,
 	COALESCE(idempotency_key, '') AS idempotency_key,
-	COALESCE(url_query_params, '') AS url_query_params
+	COALESCE(url_query_params, '') AS url_query_params,
+	created_at,updated_at,acknowledged_at
```

Comment:
Why are we adding created_at,updated_at columns to events_endpoints table?

Reply:
No, this is `fetchEventById`, unrelated to endpoints.